.github/workflows/unit_tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   skill_object_tests:
     strategy:
       matrix:
-        python-version: [ 3.7, 3.8, 3.9, '3.10' ]
+        python-version: [ 3.8, 3.9, '3.10', '3.11' ]
```

Comment:
Do you need to support 3.8 still? It'd also EoL

Reply:
I missed that in #540.. I created #542 and assigned myself to address that with 3.12 support

Reply:
Don’t forget 3.13!